department-of-veterans-affairs/va.gov-cms
# Vagov 1316 migrate updated date

To test:
1. Run migration to Import basic pages.
2. Go to content screen and verify that "Updated" date matches "Last updated" date near bottom of screen on corresponding va.gov page.
